modules/page_base.py

```diff
@@ -64,6 +64,20 @@ def __init__(self, driver, **kwargs):
                 manifest_name += f"_{char.lower()}"
         self.load_element_manifest(f"./modules/data/{manifest_name}.components.json")
 
+    _xul_source_snippet = (
+        'xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"'
+    )
+
+    def ensure_chrome_context(self):
+        """Make sure the Selenium driver is using CONTEXT_CHROME"""
+        if self._xul_source_snippet not in self.driver.page_source:
+            self.driver.set_context(self.driver.CONTEXT_CHROME)
+
+    def resume_content_context(self):
+        """Make sure the Selenium driver is using CONTEXT_CONTENT"""
+        if self._xul_source_snippet in self.driver.page_source:
+            self.driver.set_context(self.driver.CONTEXT_CONTENT)
```

Comment:
I know this was just copy-paste from above, but&hellip; why is one `ensure_*` and the other is _`resume-*`_ prefix?

Reply:
This was because it was only used in a Browser Object model, where `CONTEXT_CHROME` would have been the preferred context, but now that it's in the BasePage, it might need renaming. Thanks!